### PR TITLE
Whois target member should be message author if no member specified + other improvements

### PR DESCRIPTION
**Pull request by Wolftallemo**
RoVer should return the message author's profile should they not specify a member. It should also not attempt to look up bots. As an added bonus, double quotes will now display correctly, and all @ s will be spaced (whereas replace was only spacing the first occurence, allowing someone to mention regardless)

### CODE REVIEW

File: src/commands/rover/WhoisCommand.js [hunk 1]
```diff
@@ -31,11 +31,16 @@ class WhoisCommand extends Command {
...
-    const member = args.member
+    let member = args.member
+    // If no member is specified the member is the message author
+    if (!member) member = msg.member
     let data = {}
     if (member) { // If the member specified exists,
       const editMessage = await msg.reply(`:mag: Looking up ${member.displayName.replace(/@/g, '')}`)
 
+      if (member.user.bot) {
+        return editMessage.edit(':robot: RoVer cannot look up bots.')
```
Comment (evaera) [inline src/commands/rover/WhoisCommand.js:42]: Just return. no reply

File: src/commands/rover/WhoisCommand.js [hunk 2]
```diff
@@ -119,8 +124,10 @@ class WhoisCommand extends Command {
         }
 
         // Add a space after any @ symbols to prevent tagging @everyone, @here, and @anything else Discord adds
-        bio = bio.replace('@', '@ ')
+        bio = bio.replace(/(@)/gm, '@ ')
 
+        // Double quotes appear as &#34; so fix that
+        bio = bio.replace(/(&#34;)/gm,'"')
```
Comment (evaera) [inline src/commands/rover/WhoisCommand.js:130]: Nice

File: src/commands/rover/WhoisCommand.js [hunk 3]
```diff
@@ -31,11 +31,16 @@ class WhoisCommand extends Command {
   // This is probably the worst file in the entire project, so prepare yourself.
   // TODO: DRY this up and make the method that gets user data a method in DiscordMember
   async fn (msg, args) {
-    const member = args.member
+    let member = args.member
+    // If no member is specified the member is the message author
+    if (!member) member = msg.member
```
Comment (evaera) [inline src/commands/rover/WhoisCommand.js:36]: Why is this behavior wanted like who wants this i dont

**Review by evaera — CHANGES_REQUESTED** [review]

File: src/commands/rover/WhoisCommand.js (hunk 1 above)
Comment (Wolftallemo) [reply src/commands/rover/WhoisCommand.js:42]: Wouldn't that leave the first message (giving the appearance the command is broken)?

File: src/commands/rover/WhoisCommand.js (hunk 3 above)
Comment (Wolftallemo) [reply src/commands/rover/WhoisCommand.js:36]: We (briefly) talked about this in the discord server, do you still want this removed? Or should the user be prompted for a member if they don't supply one? I don't believe no response at all is a good idea.

File: src/commands/rover/WhoisCommand.js (hunk 1 above)
Comment (evaera) [reply src/commands/rover/WhoisCommand.js:42]: No message

File: src/commands/rover/WhoisCommand.js (hunk 3 above)
Comment (evaera) [reply src/commands/rover/WhoisCommand.js:36]: M

File: src/commands/rover/WhoisCommand.js (hunk 1 above)
Comment (Wolftallemo) [reply src/commands/rover/WhoisCommand.js]: [Latest commit](https://github.com/evaera/RoVer/pull/292/commits/c0f3a8bbc1bb75a91c09be23dd662453cb3ff113) should do it

**Comment by evaera** [comment]
Please resolve merge conflicts

**Review by evaera — APPROVED** [review]

**Pull request merged**